luigi/format.py

```diff
@@ -499,6 +503,72 @@ def pipe_reader(self, input_pipe):
     def pipe_writer(self, output_pipe):
         return OutputPipeProcessWrapper(['bzip2'], output_pipe)
 
+
+class DirectoryFormat(Format):
+    input = 'bytes'
+    output = 'dir'
+
+    # on osx/other env we have different tools, we need the way to overload
+    # should be moved into general configuration or ToolsConfig class.
+    gnu_split = 'split'  # gnu split required (gsplit on OSX)
+    gnu_cat = 'cat'
```

Comment:
not exactly. there are 2 versions : gnu , extended ( with --additional-suffix) and BSD version ( really simple one)